compiler-core/src/javascript/tests/type_alias.rs

```diff
@@ -23,3 +23,33 @@ pub opaque type OpaqueType {
 "#,
     );
 }
+
+#[test]
+fn import_indirect_type_alias() {
+    assert_ts_def!(
+        (
+            "wibble",
+            "wibble",
+            r#"
+pub type Wibble {
+  Wibble(Int)
+}
+"#
+        ),
+        (
+            "wobble",
+            "wobble",
+            r#"
+import wibble
+pub type Wobble = wibble.Wibble
+"#
+        ),
+        r#"
+import wobble
+
+pub fn foo(x: wobble.Wobble) {
```

Comment:
No "foo" please, it's a morbid WW2 reference

Reply:
Changed to "main"